### ISSUE TITLE
Local video shows black screen on android

### ISSUE THREAD

**Issue by Debrup-nitdgp** [TheWidlarzGroup/react-native-video#1381]
### Current behavior
When I am rendering a Video from my local system it shows a black screen only when my device is unplugged, I have also tried bundling my JS code but it still shows black screen, but when I am connected to my laptop eveything runs fine, its pretty weird problem

### Reproduction steps
My simple code snippet

            <View style={styles.container}>
                <Video source={require('./welcomescr.mp4')}   // Can be a URL or a local file.
                    ref={(ref) => {
                        this.player = ref
                    }}
                    hideShutterView = {true}
                    style={styles.backgroundVideo} />
            </View>

### Expected behavior
Video should play when I am unplugged

### Platform
Which player are you experiencing the problem on:

* Android ExoPlayer

**Comment by tarunmehta-quovantis** [comment]
same here , Did you find any solution?
it crashes on android when use with **rate** props on video.

**Comment by LG1710** [comment]
I have the same problem

**Comment by ironworkshop** [comment]
Any updates on this? Im having the same issue. Online video content is fine but local videos produce a black screen.

**Comment by godza** [comment]
Having the same problems on IOS and simulator

**Comment by mikeyamato** [comment]
I've come here to express the same as OP. 

I've tested my app on the following 3 devices after building out the app and loading them onto the phones (ie. not executing `react-native run-ios` or `react-native run-android`). iPhone X, Samsung Galaxy S8+, Motorola moto e5 play. The phone I'm getting a black screen is the Motorola.

What's weird is that the app utilizes RNV in two separate sections of my app. The first section has no issue, even for the Motorola. It's the second section that's the problem.

**Comment by cagataykapuagasi** [comment]
Same issue android 8.1

**Comment by romainbraun** [comment]
In my case I fixed it by using the old Android Media Player, replacing:
```
include ':react-native-video'
project(':react-native-video').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-video/android-exoplayer')
```
with
```
include ':react-native-video'
project(':react-native-video').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-video/android')
```

**Comment by gsunsnackv** [comment]
I used to run into stucked black screen all the time. The most strange one I experience is https://github.com/react-native-community/react-native-video/issues/1538

My feeling is this project manage the native video player instances very poorly (especially on Android). The project frequently mess things up when you have multiple native video player instances. In our project, we had to do a lot of difficult and hard things to ensure that we only have one instance of a screen that have the <Video> tag, thus only one native video player instance, then it won't stuck.

**Comment by CHaNGeTe** [comment]
Please test with 4.4.4.
If still not works I will reopen the issue

**Comment by Kulsoom-1** [comment]
Still getting  black screen  issue in 4.4.5 as well, any update on this?

**Comment by fangasvsass** [comment]
any update? sam issue

**Comment by victoriaSh** [comment]
Had the same issue. 
Everything worked fine but suddenly all videos became black on Xiaomi Poco M3. No any "onError" and "onLoad" calls, no errors in logcat. 
I've tried "gradlew clean", restarted everything, even made a fresh simple app only with single video screen. Nothing helped. But when I tried to install the same app on my old Nexus 5 - everything worked perfectly. 
I guess that there is Xiaomi MIUI optimization problem. After uninstalling app, switching off all MIUI optimization and unnecessary settings in developer menu, clearing build cache everything works fine.
